stumpy/scraamp.py

```diff
@@ -31,6 +31,63 @@ def _compute_PI(
     I,
     excl_zone,
 ):
+    """
```

Comment:
Ohh, wow! The entire docstring was missing here?!? 😸  Ooops!

Reply:
Yeah :)